app/classifier/index.cjsx

```diff
@@ -5,6 +5,7 @@ ClassificationSummary = require './classification-summary'
 {Link} = require 'react-router'
 tasks = require './tasks'
 drawingTools = require './drawing-tools'
+{session} = require '../lib/session'
```

Comment:
Ok - just wondering if this will fit @vrooje and @alexbfree 's uses cases where they want a sticky session id to track the user over time? Could we use a expiring local storage key or something?